custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
-
 const introduction = {
-  title: "YOUR NAME",
-  text: "And a brief description of you",
+  title: "RAIA",
+  text: "Software Developer • Not a Robot",
 };
 
 const description = {
-  text: faker.lorem.paragraph(),
+  text: `Bringing exploratory mindset to technical leadership,
+        with six years of experience in software development and computational research`,
 };
 
 const projects = [
   {
-    title: faker.lorem.words(2),
-    text: faker.lorem.words(5),
+    title: "Software development",
+    text: "Full stack development description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
+        title: "Portfolio",
+        text: "A portfolio project to show my skills and existing projects",
+        link: "https://github.com/raiaqt/betterai-portfolio",
         year: "2024",
         github: true,
         image: "placeholder.png",
       },
     ],
   },
   {
-    title: faker.lorem.words(3),
-    text: faker.lorem.words(4),
+    title: "Competitive programming",
+    text: "Competitive programming description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
+        title: "Kattis submissions",
+        text: "Solutions to algorithm and data structure problems on ",
+        year: "2023",
+        link: "https://github.com/raiaqt/kattis-submissions",
         github: true,
         image: "placeholder.png",
       },
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
+        title: "OnlineJudge submissions",
+        text: "Solutions to algorithm and data structure problems on ",
+        year: "2023",
+        link: "https://github.com/raiaqt/onlinejudge-submissions",
         github: true,
       },
     ],
   },
   {
-    title: faker.lorem.words(3),
-    text: faker.lorem.words(4),
+    title: "Writing",
+    text: "Writing description",
     works: [
       {
-        title: faker.word.noun(),
-        text: faker.lorem.sentence(),
-        link: "/",
-        year: "2024",
-        github: true,
```

Comment:
[via ChatGPT review bot] I see that you've switched out using `faker` to populate your data to hard-coded values. While hard-coding may work for a personal project, for more dynamic or larger scale applications, you may want to consider using a database or an external database to store your data, as it can reduce clutter in your code and allow for easier updates in the future.

Also, You might want to consider making these variable names a bit more descriptive than `title` and `text`. Consider naming them something that indicates what they actually represent like `jobTitle` or `jobDescription`.

You may refer to this [Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html#naming) for naming conventions.

```javascript
const currentJob = {
  jobTitle: 'Lead Engineer',
  company: 'Samsung R&D Institute Philippines',
  jobDescription: [
    'Develops and deploys new features for multiple enterprise applications',
    'Leads technical decisions throughout the development lifecycle',
    'Mentors teammates in various topics such as data structures, algorithms, and front-end development'
  ],
};

const education = {
  degree: 'BS Biochemistry',
  school: 'University of the Philippines Manila',
  honors: ['Cum Laude'],
  yearsAttended: '2012-2016',
};
```